app/workers/medium_removal_worker.rb

```diff
@@ -3,8 +3,8 @@ class MediumRemovalWorker
 
   sidekiq_options queue: :data_low
 
-  def perform(medium_src)
-    MediaStorage.delete_file(medium_src)
+  def perform(medium_src, opts={})
+    MediaStorage.delete_file(medium_src, opts)
```

Comment:
Can we please update the method signature of the abstract & aws adapters? 
https://github.com/zooniverse/panoptes/blob/ff90c7ecd73c66c2af03895a997e44b369973d26/app/services/media_storage/abstract_adapter.rb#L23
https://github.com/zooniverse/panoptes/blob/ff90c7ecd73c66c2af03895a997e44b369973d26/app/services/media_storage/aws_adapter.rb#L60

Reply:
Sure thing. I have marked the `opts` param as unused in the aws adapter using an underscore:
https://github.com/zooniverse/panoptes/blob/8457f16bfc92e5153e725e37a70032cc75aaa26f/app/services/media_storage/aws_adapter.rb#L60